frontend/cypress/integration/register-page/register-page.spec.js

```diff
@@ -0,0 +1,38 @@
+import RegisterFactory from "../../factories/RegisterFactory";
+import RegisterPage from "../../pages/RegisterPage"
+context('register page', () => {
+    
+    it('Should open the registration page by clicking on the registration item', () => {
+        cy.visit(`/login`);
+        cy.get('[id="btn-register"]').click();
+        cy.url().should('be.equal', `${Cypress.config('baseUrl')}/register`);
+      });
+
+    it('Should must successfully register as a teacher', ()=> {
```

Comment:
```suggestion
    it('Should successfully register as a teacher', ()=> {
```
For this and for other places, you should use only `should` or `must`, never both together